models/User.js

```diff
@@ -31,6 +31,10 @@ User.init(
         isEmail: true,
       },
     },
+    zipcode: {
+      type: DataTypes.STRING,
```

Comment:
Should this be an integer? can you elaborate on why we're choosing to have it be a string?

Reply:
no

Reply:
Can you elaborate on why we're choosing to have it be a string???